ws2812.py

```diff
@@ -45,9 +47,12 @@ def __init__(self, ledNumber=1, brightness=100):
         # SPI init
         # Bus 0, 8MHz => 125 ns by bit, 8 clock cycle when bit transfert+2 clock cycle between each transfert
         # => 125*10=1.25 us required by WS2812
-        self.spi = SPI(0, SPI.MASTER, baudrate=8000000, polarity=0, phase=1)
-        # Enable pull down
-        Pin('GP16', mode=Pin.ALT, pull=Pin.PULL_DOWN)
+        if uname().sysname == 'LoPy':
+            self.spi = SPI(0, SPI.MASTER, baudrate=8000000, polarity=0, phase=1, pins=(None, dataPin, None))
```

Comment:
I am not familiar with LoPy, but I assume you have a pull-down ? It might be a good thing to activate it